tests/python/tests/http_server/php/index.php

```diff
@@ -178,6 +178,22 @@ public function work(string $output) {
 
         break;
     }
+} else if ($_SERVER["PHP_SELF"] === "/test_headers_sent") {
+    switch($_GET["type"]) {
+        case "flush":
+            echo (int)headers_sent();
+            flush();
+            echo (int)headers_sent();
+            break;
+        case "shutdown":
+            if ((int)$_GET['flush']) {
+                flush();
+            }
+            register_shutdown_function(function() {
+              file_put_contents("out.dat", (int)headers_sent());
```

Comment:
Try to avoid a file usage. Better to use a printing into stderr (e.g. `fprintf`) and сheck an assertion for a server log (e.g. `self.kphp_server.assert_log`).

Reply:
done